File: CustomVoice-API-Samples/CSharp/CustomVoice-API/API/DTO/Voice.cs
using Newtonsoft.Json;
using System;

namespace CustomVoice_API.API.DTO
{
    class Voice
    {
        [JsonConstructor]
        private Voice(Guid id, string name, string locale, string gender, bool isPublicVoice)
        {
            this.Id = id;
            this.Name = name;
            this.Gender = gender;
            this.Locale = locale;
            this.IsPublicVoice = isPublicVoice;
        }

        public string Name { get; private set; }


        public string Locale { get; private set; }


        public Guid Id { get; private set; }


        public string Gender { get; private set; }

        public bool IsPublicVoice { get; private set; }
    }
}

File: CustomVoice-API-Samples/Java/src/main/java/customvoice/model/Voice.java
/*
 * Speech Services API v3.0-beta1
 */

package customvoice.model;

import java.util.UUID;

import com.google.gson.annotations.SerializedName;

/**
 * Voice
 */
public class Voice {
  @SerializedName("locale")
  private String locale = null;

  @SerializedName("name")
  private String name = null;

  @SerializedName("id")
  private UUID id = null;

  @SerializedName("gender")
  private String gender = null;

  @SerializedName("isPublicVoice")
  private Boolean isPublicVoice = null;

  public String getLocale() {
    return locale;
  }

  public void setLocale(String locale) {
    this.locale = locale;
  }

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }

  public UUID getId() {
    return id;
  }

  public void setId(UUID id) {
    this.id = id;
  }

  public String getGender() {
    return gender;
  }

  public void setGender(String gender) {
    this.gender = gender;
  }

  public Boolean isIsPublicVoice() {
    return isPublicVoice;
  }

  public void setIsPublicVoice(Boolean isPublicVoice) {
    this.isPublicVoice = isPublicVoice;
  }
}
